Lacework template (#116)
buildkite/templates: lacework/readme.md

Reviewer: Build pipeline is tripping up because this is lowercase. This file should be `README.md`

Reviewer: Looks like you'll still need to rename this file for the build to pass.

Reviewer 2: This was a bit of a pain! Had to delete the file and then recreate it complete as git didn't want to pick up my changes 🤔

---

> title: Vulnerability scanning with Lacework

Reviewer: For consistencies sake, perhaps this should be more like the Snyk template? Or perhaps that should be updated to be more inline with this one. 

e.g.
```suggestion
title: Scan project using Lacework
```

<img width="336" alt="Screenshot 2024-05-15 at 1 12 54 pm" src="https://github.com/buildkite/templates/assets/656826/ec309e2d-5965-4fe5-b9f5-aeef256b5525">

> title: Software Composition Analysis with Lacework

Reviewer: **(nitpick)**: Can we use sentence casing to match others and replace "with" with "using" to match some of the other templates.

```suggestion
title: Software composition analysis using Lacework
```

description: Audit your project for security vulnerabilities using Lacework.
author: Buildkite
use_cases: ["Security", "CI", "Schedule"]
tools: ["Lacework"]
primary_emojis: [:lacework:]
---

# Scan using Lacework

This template gives you a continuous integration (CI) pipeline that scans a a sample project for security vulnerabilities using Lacework.

At a glance:

- Uses [Lacework](https://www.lacework.com/) and requires an account and API key for security scanning

> - Requires [Docker](https://docs.docker.com/get-docker/)

Reviewer: Does this pipeline use docker? I think you can drop this.

## How it works

This template:

1. Sets up the environment with Lacework

> 2. Runs Lacework over yourproject for security vulnerabilities.

Reviewer: ```suggestion
2. Runs Lacework over your project for security vulnerabilities.
```

## Next steps

After you select **Use template**, you’ll:

1. Connect your git repository.
2. Modify the template commands, environment variables, secrets as needed for your project.

> 3. [Create](https://docs.lacework.net/console/api-access-keys) and store your Lacework API token securely on your agents.

Reviewer: Perhaps it's worth calling out the environment variable names required for the [plugin](https://github.com/buildkite-plugins/lacework-buildkite-plugin) so that it's obvious which specifc env is needed? ie. `LW_API_SECRET` `LW_API_KEY`.

We do something similar for other templates 

e.g. https://buildkite.com/pipelines/templates/cd/vercel-deploy?queryID=34d9534528a9a7066ae721c0257dd0c6
